src/hostname/hostnamed.c

```diff
@@ -353,6 +359,14 @@ static int get_hardware_model(Context *c, char **ret) {
         return get_dmi_properties(c, STRV_MAKE_CONST("ID_MODEL_FROM_DATABASE", "ID_MODEL"), ret);
 }
 
+static int get_hardware_sku(Context *c, char **ret) {
+        return get_dmi_property(c, "ID_SKU", ret);
+}
+
+static int get_hardware_version(Context *c, char **ret) {
+        return get_dmi_property(c, "ID_VERSION", ret);
```

Comment:
ID_VERSION is a bit too generic for my taste. maybe rename the props ID_HWSKU and ID_HWVERSION or so?

Reply:
if you want you can drop the HW from ID_HWSKU, i.e. ID_SKU is descriptive/clear enough i'd say.

but otherwise I think this asymmetry is ok